src/openzaak/config/models.py

```diff
@@ -11,7 +11,7 @@ class NLXConfig(SingletonModel):
     directory = models.CharField(
         _("NLX directory"), max_length=50, choices=NLXDirectories.choices, blank=True
     )
-    outway = models.URLField(_("outway address"), blank=True)
+    outway = models.URLField(_("outway address"), blank=True, help_text=_("Example: http://my-outway.nlx:8080")
```

Comment:
What about using an example domain URL here?

https://www.iana.org/domains/reserved

Reply:
Not a bad suggestion - however, since outways are not running on publicly accessible domains, this might give an incorrect idea that the outway address should look like a publicly available domain? Currently we've been deploying NLX inway/outway in the `nlx` namespace in kubernetes clusters, and the port number is also very relevant, so I figured this is closer to the real situation. Thoughts?

Reply:
If it would be more clear to use an "nlx" domain, because the real-life versions will have that, then maybe that is better. If "nlx" becomes a top level domain, perhaps "example.nlx" would become a reserved name in that domain, thus maybe something like "my-outway.example.nlx"? But this is a small thing. Perhaps more important is probably to think about if "nlx" becomes a new TLD, how will that intersect with the kubernetes deployments.

Reply:
in theory, `my-outway.nlx.svc.cluster.local` should be the DNS name that makes clear it's internal in the cluster, but Open Zaak also deploys on VMs without K8s, where the hostname is rather `nlx-outway` without TLD.

In the cluster I've been experimenting with, I also found that `my-outway.nlx` did work and the `my-outway.nlx.svc.cluster.local` did not - might be a version thing.

In the end - we expect the configurator to know where the NLX outway is hosted or at least know who to ask this. I'm not sure we should worry too much about `.nlx` becoming a TLD - even in a k8s the local DNS trumps the external DNS :thinking:

Reply:
I agree, this is not a big deal -- if there were an obvious "better" thing, then great, but if not, we move on.